docs/k8s/getting-started-gwapi.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Getting Started - Kubernetes Gateway API
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import K8sIngressInstallExample from "/examples/k8s/install-ingress.mdx";
+import K8sGatewayAPIInstallExample from "/examples/k8s/install-gateway-api.mdx";
+
+import K8sIngressModifyHeadersExample from "/examples/k8s/http-request-headers.mdx";
+import K8sGatewayModifyHeadersExample from "/examples/k8s/http-request-headers-gateway.mdx";
+
+import K8sIngressRedirectExample from "/examples/k8s/http-request-redirect.mdx";
+import K8sGatewayRedirectExample from "/examples/k8s/http-request-redirect-gateway.mdx";
+
+import StaticDomainKubernetesExample from "/examples/k8s/http-static-domain.mdx";
+import StaticDomainKubernetesGatewayExample from "/examples/k8s/http-static-domain-gateway.mdx";
+
+import StaticDomainKubernetesPathExample from "/examples/k8s/http-static-domain-path.mdx";
+import StaticDomainKubernetesGatewayPathExample from "/examples/k8s/http-static-domain-gateway-path.mdx";
+
+# Kubernetes Ingress Controller
+
+## Overview
+
+The Kubernetes Operator allows access to k8s Services using the new [Gateway API](https://gateway-api.sigs.k8s.io/).
+This implementation includes CRDS for a Gateway as well at HTTPRoute.
+Additional routes, such as TLSRoute and TCPRoute will be added at a later date when they move from exerimental to stable.
+
+## Install
+
+Installing the ngrok Operator is easy using Helm.
+
+<K8sGatewayAPIInstallExample />
+
+<br />
+
+## Create an edge with a domain
+
+This would create an Edge on with the Domain `example.ngrok.app` that routes traffic to the Kubernetes service `example-service` running on port 80.
+
+<StaticDomainKubernetesGatewayExample />
+
+## Route Traffic based on a path
+
+This would create an Edge on with the Domain `example.ngrok.app` that routes traffic from example.ngrok.app/foo to the Kubernetes service `example-service` running on port 80.
+
+<StaticDomainKubernetesGatewayPathExample />
+
+## Modify Headers
+
+This would modify the headers
+
+<K8sGatewayModifyHeadersExample />
+
+## Redirect Traffic
+
+This would redirect a request to another URL
+
+<K8sGatewayRedirectExample />
+
+## Pricing
+
+The ngrok Kubernetes Gateway API is available to all ngrok users at no
```

Comment:
the ngrok k8s operator is available to all users, which allows users to work with the k8s gateway api.

Reply:
I modified this to clarify that the operator is free, but the resources used for the Gateway API are what includ the cost.